hybrid/ddc_bounds.c

```diff
@@ -0,0 +1,21 @@
+#include "../include/common.h"
+#include "include/utils.h"
+#include <stdint.h>
+#include <stdlib.h>
+#if !defined(__aarch64__)
+#error "This example is Morello only"
+#endif
```

Comment:
I'd put that on the implementation of `write_ddc`, since that's the only Morello-specific part (and could be adapted for other CHERI implementations, potentially).

Reply:
I still have that `dcc_get` issue so I have to also read from the DDC using `read_ddc` (*morello only*).